streamlit_prophet/lib/dataprep/format.py

```diff
@@ -558,6 +559,7 @@ def prepare_future_df(
             freq=dates["forecast_freq"],
         )
         future = pd.DataFrame(future_dates, columns=["ds"])
+        future = add_cap_and_floor_cols(future, params)
```

Comment:
Could we move this line out of the "else" statement, so that the ``add_cap_and_floor_cols`` function is called in any case?

Reply:
And also, would it be possible to add the text below in the parameter's part of the docstring of ``prepare_future_df`` and ``make_future_df`` functions?

params : Dict
      Dictionary containing all model parameters

Thanks a lot!